nahusenayElias/Team_4
# DRD-103: styling Services page

## Related ticket
[DRD-103](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1?selectedIssue=DRD-103)

## In this PR: 
- Modified styles in Layout.jsx (increased the width)
- Added "cards" for Services page, styled buttons, added a Get in touch button in the Services page and Back to Services button to individual Service pages.

## Testing instructions: 
- checkout branch
- See the changes in the Services page and test that buttons work
<img width="1223" alt="Screenshot 2024-12-10 at 10 35 34 1" src="https://github.com/user-attachments/assets/11d5f91f-27a9-4c58-8ea2-5388934b49b4">


[DRD-103]: https://edu-team4-react24k.atlassian.net/browse/DRD-103?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
